internal/access/deviceflow.go

```diff
@@ -20,12 +20,18 @@ func FindDeviceFlowAuthRequest(ctx RequestContext, deviceCode string) (*models.D
 		return nil, err
 	}
 
-	// TODO: do this in the data function
+	// TODO: do these lookups in the data function when data.GetDeviceFlowAuthRequest
+	// is converted to SQL
 	if dfar.AccessKeyID > 0 {
 		dfar.AccessKey, err = data.GetAccessKey(ctx.DBTxn, data.GetAccessKeysOptions{ByID: dfar.AccessKeyID})
```

Comment:
Not part of this change but it seems a bit odd that the `DeviceFlowAuthRequest` has a separate `AccessKeyID` and `AccessKey`